src/main/java/com/example/bak/community/application/dto/CommunityResult.java

```diff
@@ -0,0 +1,26 @@
+package com.example.bak.community.application.dto;
+
+import com.example.bak.community.domain.Community;
+
+public class CommunityResult {
+
+    public record ResourcePath(Long communityId) {
+
+        public static ResourcePath from(Community community) {
+            return new ResourcePath(community.getId());
+        }
+    }
```

Comment:
네이밍을 조금 더 도메인적으로 할 수 있을것 같은데(사실 ResourcePath의 postfix 의 부분 형태만 가지고 오기 때문에 CommunityId라는 이름 네이밍으로 해도 될거 같아요 그냥 그리고 Long value 로 하면 어떨까요?

```suggestion
    public record CommunityId(Long value) {

        public static CommunityId from(Community community) {
            return new CommunityId(community.getId());
        }
    }
```

Reply:
좋은데요? 좀 더 메서드 느낌나고 이게 나을 거 같습니다! 반영할게요~